pkg/config/config_test.go

```diff
@@ -617,21 +556,21 @@ func TestNewOrgConfigGroup(t *testing.T) {
 		}
 	},
 	"values": {
-		"AnchorPeers": {
+		"Endpoints": {
 			"mod_policy": "Admins",
-			"value": "CgkKBWhvc3QxEHs=",
+			"value": "Cg1sb2NhbGhvc3Q6MTIz",
```

Comment:
We figured out this was because we were using protolator on an arbitrary ConfigGroup. protolator is unable to determine how to decorate the ConfigGroup if we don't explicitly say what type of ConfigGroup it is. By wrapping it with a DynamicOrdererOrgGroup in the tests it now properly decodes the values.